README.md

```diff
@@ -171,6 +172,13 @@ An example of a micro-ROS application using PlatformIO is available [here](https
 docker pull microros/micro_ros_static_library_builder:galactic
 docker run -it --rm -v $(pwd):/project --env MICROROS_LIBRARY_FOLDER=extras microros/micro_ros_static_library_builder:galactic
 ```
+
+Optionally a specific single target can be built using the `-p <STATIC_TARGET>` argument like this:
```

Comment:
I would rename STATIC_TARGET to LIBRARY_TARGET and reference all the available options: https://github.com/micro-ROS/micro_ros_arduino/blob/08c0e7d8b1f5d1e6d7e1c36bdb267c208066a324/extras/library_generation/library_generation.sh#L13-L24

Reply:
done